README.md

```diff
@@ -118,6 +118,8 @@ Once Media SDK was built with ITT support, enable it in a runtime creating per-u
 ```sh
 Output=0x10
 ```
+# Known limitations
+Windows build contains only dispatcher and samples. MediaSDK library DLL is provided with Windows GFX driver.
```

Comment:
The problem is that we don't have dispatcher yet.

Reply:
Fixed